### ISSUE TITLE
Thaumic Wand is having issues with RAM ... PLEASE us thanks...

### ISSUE THREAD

**Issue by Xiantrius** [Zoko061602/Thaumic-Wands#40]
I use MC v1.12.2 forge
My PC

Mainboard ASUS ROG Strix B550-F Gaming Mainboard Sockel AM4
AMD Ryzen 9 5950x 32 threads and 16 cores
Radeon RX 6700XT
128GB RAM DDR4
System: Windows 10

https://github.com/orgs/community/discussions/43452

The investigation took more than 1 day to find the error and I hope that you will be gracious to us and help us.

I test this with 3 java versions 1.8.351, 1.8.333, 1.8.251
I test 2 versions Launcher's

Then i found this problem...

When I see the items from the mod Thaumic Wands, the RAM increases more and more over time.
All my friends who are on my server have the same problem. The problem also occurs in single play.
I ask you to make a fix so that the memory of your mod does not always fill up as soon as you see items from the mod.
I actually like playing your mod and the people who play the mod with me would also be happy to get a fix.

The error is reproducible.

1. Start a single game or multiplayer with this mod.

2. If you haven't already done so, switch the full screen to windowed mode.

3. Start the task manager and look for javaw.

4. If you use the mod jei give a Thaumic Wands, or in creative mode get one of the items.

5. Now pay attention to javaw and look at the memory value.

6. Delete the item or put it in any chest so that the Thaumic Wands items are no longer visible.

7. Repeat point 5.

![Task-Manager 04 01 2023 19_27_14](https://user-images.githubusercontent.com/42416073/210850648-ae7d9778-38d0-4f53-9718-a36f5f677772.png)

Conclusion:
I was wondering why my powerful PC has already frozen twice because Minecraft was open over time because of this mod. I really hope you can find the cause and fix it.
Unfortunately, the log doesn't spit anything out. Likewise, there is no crash, but the task manager shows it. the biggest record was more than 100GB until the system froze. Please take my words seriously and don't doubt them.

Please help us.

**Comment by InterPlay02** [comment]
Oh... So that's why my game just crashed two times today.
Both crashes was telling me the same thing: the system ran out of memory.

Dude... this is bad. As you have said, I like playing with this mod.
But is sad when just by holding one of it's wand will cause (almost) my entire PC to crash.

(The game closes, the CurseForge App closes, Overwolf app freezes and my Windows Explorer also crashes and starts to get crazy)

Only after I close and reopen everything, things comes back to normal.

Well.. now we can only hope if one day this will be fixed.

**Comment by Xiantrius** [comment]
@InterPlay02 
As long as the problem is not solved, I recommend that you delete it. I know that's not nice, but safety comes first in the game and I did that and deleted the mod and I did that very reluctantly. Unfortunately the developer doesn't even get in touch here, really sad.

best regards
Xiantrius

**Comment by Zokonius** [comment]
Sorry for the long wait, the problem should be fixed and is on curseforge now (version 1.2.7).

**Comment by Xiantrius** [comment]
> Sorry for the long wait, the problem should be fixed and is on curseforge now (version 1.2.7).

wow we have to wait 1 year until the programmer reports on this mod. Well, nothing but thanks for taking a look and fixing the error at the same time.
Many don't answer at all or flee because they don't feel like it anymore. To be honest, I had given up hope.

My Test result:
I also confirm that the RAM problem with the Thaumic Wands mod has indeed been fixed. Thank you :)

best regards
Xiantrius

**Comment by Ushupiuck** [comment]
well, if this is solved, you may as well close the issue, @Xiantrius
